src/ontology/mondo-edit.obo

```diff
@@ -106138,15 +106144,9 @@ is_a: MONDO:0004953 {source="EFO:1000047", source="NCIT:C5161"} ! invasive ducta
 
 [Term]
 id: MONDO:0006051
-name: postweaning multisystemic wasting syndrome
-def: "Pig disease caused by porcine circovirus type 2 (PCV2). Most representative symptoms include wasting, unthriftness, paleness of the skin, respiratory distress, diarrhea and sometimes icterus." [EFO:1000048]
-comment: Reason: duplicate. This will be merged with MONDO:0025506 porcine postweaning multisystemic wasting syndrome
-subset: obsoletion_candidate
-xref: EFO:1000048 {source="MONDO:equivalentTo"}
-is_a: MONDO:0005583 {source="EFO:1000048"} ! non-human animal disease
-relationship: in_taxon NCBITaxon:9823 {source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648"} ! Sus scrofa
-property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/5294 xsd:anyURI
-property_value: IAO:0006012 "2022-11-01" xsd:string
+name: obsolete postweaning multisystemic wasting syndrome
+is_obsolete: true
+replaced_by: MONDO:0025506
```

Comment:
for the obsoleted terms, could you please the term tracker URL and the annotation 'has obsolescence reason' terms merged?

thanks!